src/cli/commands/dev.ts

```diff
@@ -141,3 +130,31 @@ export class Dev implements Command {
       `
   }
 }
+
+function getTranspiledStartModule(layout: Layout.Layout, reflectionMode: boolean) {
```

Comment:
Should this be in the start module lib?

Reply:
Thought about it, but found that it wasn't really helpful. This function is there just to prevent from repeating the numerous parameters needed to the two functions, twice. (once when for what we pass to the watcher, and once in the dev plugin to refresh the start-module after every file change).
This is really just a very local helper to keep the code dryer

![image](https://user-images.githubusercontent.com/25233379/82216172-d92b9b80-9918-11ea-9b62-e5c9a645f696.png)